lib/workers/global/index.ts

```diff
@@ -63,10 +63,12 @@ function checkEnv(): void {
   const range = pkg.engines!.node!;
   const rangeNext = pkg['engines-next']?.node;
   if (process.release?.name !== 'node' || !process.versions?.node) {
-    logger.warn(
-      { release: process.release, versions: process.versions },
-      'Unknown node environment detected.'
-    );
+    if (!process.env.RENOVATE_X_IGNORE_NODE_WARN) {
```

Comment:
we need same change on line 81 below otherwise we still warn on next range mismatch.

Reply:
Done